hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -61,8 +61,9 @@
   private String tableName;
 
   @CommandLine.Option(names = {"--with-keys"},
+      required = true,
```

Comment:
Thanks to @errose28 ’s review, I think it’s better to change `required` to `false` here. The purpose of using `false` is to allow users to make choices according to their needs. What do you think?

Reply:
> Thanks to @errose28 ’s review, I think it’s better to change `required` to `false` here. The purpose of using `false` is to allow users to make choices according to their needs. What do you think?

According to [Picocli's package documentation](https://picocli.info/apidocs/picocli/CommandLine.Option.html#required--), the default value of this `required` element is `false`, so I think you just need to delete this line.

![image](https://user-images.githubusercontent.com/14295594/146494383-7717515a-4bec-4e07-8058-d7a69179d894.png)

Reply:
Thanks to @cku328  for the documentation.
This is a discussion, if `required` is set to `false`, I will delete this line.

Reply:
My 2 cents: I don't think boolean flags should ever be required, as they only have two possible values.  Usually they should default to `false` (assuming the variable that gets set matches their wording), so that you only have to specify it if you want to set it to `true`.  If we want the default behavior to be including keys in output, the flag would better be worded as `--omit-keys`.  Default value would be `true` if the variable is still `withKeys`, or `false` if the variable is renamed and its usage flipped (`if (withKeys)` -> `if (!omitKeys)`).

Reply:
Thanks again @sky76093016 for the patch.  Can we resolve this minor issue about `required = true` parameter?

Reply:
@sky76093016 do you mind if I make this minor tweak to your change?